spdx/writers/jsonyamlxml.py

```diff
@@ -370,6 +370,9 @@ def create_snippet_info(self):
             snippet_object["licenseInfoFromSnippet"] = list(
                 map(self.license, snippet.licenses_in_snippet)
             )
+            byte_range = {"endPointer": {"offset": snippet.byte_range[1], "reference": snippet_object["fileId"]},
```

Comment:
Instead of referencing `snippet_object["fileId]"` four times, you could extract `self.spdx_id(snippet.snip_from_file_spdxid)` into a variable in line 369. This would also avoid using a property of the object that we are creating in this method, which I found somewhat confusing when first reading this

Reply:
I extracted `self.spdx_id(..)` in a variable, not in line 369 but I'm not sure if this line mismatch is due to other changes. Or did you suppose to set this variable in the `__init__` function? This would not work as this variable is different per snippet and `parse_snippets` iterates through all snippets, so the `SnippetParser` isn't initialized per Snippet.